frontend/src/state/backendApi.ts

```diff
@@ -234,10 +237,17 @@ const apiStore = {
                     this.MessageSearchPhase = msg.phase;
                     break;
 
+                case 'progressUpdate':
+                    this.MessagesBytesConsumed = msg.bytesConsumed;
+                    this.MessagesTotalConsumed = msg.messagesConsumed;
+                    break;
+
                 case 'done':
                     this.MessagesElapsedMs = msg.elapsedMs;
                     // this.MessageSearchCancelled = msg.isCancelled;
                     this.MessageSearchPhase = "Done";
+                    this.MessagesBytesConsumed = 0;
```

Comment:
better to reset values to zero when a new message search starts (and/or when the websocket.closed event happens)